Add CITATION.cff

simleo opened the pull request:
Adds [CITATION.cff](https://citation-file-format.github.io) to the repo. This is [now supported by GitHub](https://twitter.com/natfriedman/status/1420122675813441540).

All should review this, especially to check if I got the ORCIDs right.

At some point we should review and single-source all this information. This file looks like the best source for authors, since it contains more information and it's structured; OTOH, the release date should probably be generated upon tagging.

stain commented:
+1 Checked all ORCIDs match listing on https://www.researchobject.org/ro-crate/community.html except @albangaignard which anyway match http://albangaignard.github.io/about/

The DOI is the 0.4.0 release, meaning we would have to update it on each release - and it would be off-by-one within each release as the DOI is made automatically afterwards. Should we change to the version-less DOI https://doi.org/10.5281/zenodo.3956493 so we don't have to change it?  Or as we have to update `version` anyway before a release, just bite the bullet that the DOI will be wrong at tagging time?

I note that for instance [cff-converter-python](https://github.com/citation-file-format/cff-converter-python/blob/1.3.2/CITATION.cff) uses the version-less DOI.

BTW - is the `CITATION.cff` accessible from pip? We could add it somehow in setup.py so that this is accessible also outside github?

simleo commented:
> The DOI is the 0.4.0 release, meaning we would have to update it on each release - and it would be off-by-one within each release as the DOI is made automatically afterwards. Should we change to the version-less DOI https://doi.org/10.5281/zenodo.3956493 so we don't have to change it? Or as we have to update `version` anyway before a release, just bite the bullet that the DOI will be wrong at tagging time?

Changed to the versionless DOI. Seems more appropriate for something that's supported by a repository widget, since it reflects the ongoing development version. For instance, we still had to add @ilveroluca to the authors and now I added him in 84cd65c, however he wasn't an author as of release 0.4.0.

> BTW - is the `CITATION.cff` accessible from pip? We could add it somehow in setup.py so that this is accessible also outside github?

Added `CITATION.cff`, and `LICENSE` which was missing.
